### ISSUE TITLE
please provide pretrained weights of train2.py

### ISSUE THREAD

**Issue by Harsh2040** [andabi/deep-voice-conversion#46]
Hi guys, 

I'm suffering through the loss fluctuations in NET2 architecture and I'm training in on arctic dataset. 
if anybody fine tuned net2 architecture on any dataset then please add it up here.
Thanks in advance.

**Comment by YashBangera7** [comment]
**##Urgent##**
Did you train the model and do you have the pretrained weights. please help. Im stuck . Need urgently

**Comment by Amey1109** [comment]
> **##Urgent##**
> Did you train the model and do you have the pretrained weights. please help. Im stuck . Need urgently

Please Guys Do help its urgent if anybody has trained the net2 on any dataset please upload
